### ISSUE TITLE
Gracefully handle IntegrityError on FCMDeviceAuthorizedViewSet

### ISSUE THREAD

**Issue by d1p** [xtrinch/fcm-django#61]
Right now if I send a request with the `id` parameter and if the id already exists the server just throws a 500 error. Where either `id` field should be `read_only` or it should return an HTTP client error with the error message in the body.

**Comment by xtrinch** [comment]
Be more specific. Request where and what is the error returned with DEBUG = True

**Comment by lukeburden** [comment]
Why is it even possible to change the `id` field? Is there a use-case for that?

**Comment by d1p** [comment]
Error:
```
duplicate key value violates unique constraint "fcm_django_fcmdevice_pkey" DETAIL: Key (id)=(1) already exists.
```

upon having a duplicate key it should result in a 400 error with a meaningful message so that frontend developers can work with it, Right now it's just creating an error 500.

**Comment by d1p** [comment]
Alright, I will submit a PR and hopefully, it will get approved.

**Comment by xtrinch** [comment]
This project is in desperate need of maintainers as I am very short on time. But I will do my best to review.

**Comment by xtrinch** [comment]
Fix merged in #72
